src/refactor/interactiveHandler.py

```diff
@@ -0,0 +1,112 @@
+
+from system import services
+from inputParser import inputParser
+from receiveParser import receiveParser
+from embedCSP import embedPacket
+import time
+
+class interactiveHandler:
+    def __init__(self):
+        self.services = services
+        self.inParser = inputParser()
+
+        # TODO: This is bad, make it good when fixing inputParser
+        self.appIdx = 0
+        self.serviceIdx = 2
+        self.subserviceIdx = 4
+
+    def getTransactionObject(self, command : str, networkHandler):
```

Comment:
Thanks for this! This just made writing the communications script much easier when handling different types of commands